engine/runtime-instrument-common/src/main/java/org/enso/interpreter/service/ExecutionService.java

```diff
@@ -178,16 +178,6 @@ public void execute(
     var pending =
         submitExecution(
             () -> {
-              SourceSection src;
```

Comment:
This change is _strictly speaking_ not necessary as I have fixed the underlying bug.
But the check is also obsolete - we check the source earlier as covered by our test suite.